examples/amp-story/interactive_slider.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <meta name="description" content="This is the AMP Boilerplate." />
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link
+      rel="preload"
+      as="script"
+      href="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+    />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+      async
+      custom-element="amp-story"
+      src="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-story-interactive"
+      src="https://cdn.ampproject.org/v0/amp-story-interactive-0.1.js"
+    ></script>
+    <!-- Import other AMP Extensions here -->
+    <style amp-custom>
+      /* Add your styles here */
+      .title {
+        background-color: white;
+        border-radius: 60px;
+        padding: 40px;
+      }
+      h1 {
+        color: black;
+        font-size: 30px;
+        margin-bottom: 10px;
+        text-align: center;
+        font-family: 'Poppins';
+      }
+      p {
+        font-family: 'Poppins';
+        font-size: 15px;
+      }
+      .bottom {
+        position: absolute;
+        top: 80%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        width: 22em;
+      }
+    </style>
+    <style amp-boilerplate>
+      body {
+        -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      }
+      @-webkit-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-moz-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-ms-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-o-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
```

Comment:
nit: Please use the one-line unformatted amp-boilerplate styling from [https://amp.dev/boilerplate/](https://amp.dev/boilerplate/)